src/pages/tBTC/Bridge/Minting/MakeDeposit.tsx

```diff
@@ -119,56 +117,59 @@ const MakeDepositComponent: FC<{
     useTbtcState()
 
   return (
-    <Box mx={{ base: 0, lg: 10 }}>
+    <Box mx={{ base: 0, xl: 10 }}>
       <BridgeProcessCardTitle
         onPreviousStepClick={onPreviousStepClick}
         badgeText="2/3"
         title="Make your BTC deposit"
+        description={
+          <>
+            Use this generated address to send minimum{" "}
+            <Text as="span" color="white">
+              0.01 BTC
```

Comment:
But we could use `&nbsp;` here:

```suggestion
              0.01&nbsp;BTC
```

Reply:
Ref commit: 25a9c7c26a385ee187058fe261fcb4d18a28048a